tests/hybrid/test_dispatch.py

```diff
@@ -626,6 +626,26 @@ def test_hybrid_dispatch_heuristic(site):
     assert sum(hybrid_plant.battery.dispatch.charge_power) > 0.0
     assert sum(hybrid_plant.battery.dispatch.discharge_power) > 0.0
 
+def test_hybrid_dispatch_baseload_heuristic(site):
+    dispatch_options = {'battery_dispatch': 'baseload_heuristic',
+                        'grid_charging': False}
+    wind_solar_battery = {key: technologies[key] for key in ('pv', 'wind', 'battery')}
+    baseload_limit_kw = float(interconnect_mw * 0.75 * 1000)        # in kilowatts
+    baseload_percent = 90.0
+    simulation_options = {'control':{'baseload':{'baseload_limit':baseload_limit_kw, 'baseload_percent': baseload_percent}}}
```

Comment:
In this test, we require the user to set both the `battery_dispatch` and the `control` dictionary. If we move these inputs into `dispatch_options` at least they will be defined right next to each other. We should also add a check to make sure the user has provide the correct keys if a specific `battery_dispatch` is selected.